data.html

```diff
@@ -3,4 +3,87 @@
 title: Understanding the Data
 layout: default
 ---
-<h1>Understanding Data</h1>
\ No newline at end of file
+<main class="ds-base ds-l-container">
+  <a class="ds-c-skip-nav" href="#content">Skip to main content</a>
+  <div class="ds-l-row ds-u-fill--primary-darker">
+    <div class="ds-l-col--12 ds-u-text-align--right ds-u-font-size--display ds-u-color--white">
+      Understanding the Data
+    </div>
+    <div class="ds-l-col--12 ds-u-text-align--right ds-u-font-size--h1 ds-u-color--white">
+      The User Guide for ACOs in Production
+    </div>
+  </div>
+  <div class="ds-l-row ds-u-padding-top--2">
+    <aside class="ds-u-margin-bottom--4 ds-u-md-margin-bottom--0 ds-l-md-col--4 ds-l-lg-col--3">
+      <ul class="ds-c-vertical-nav">
```

Comment:
Looks like we are missing:
- Understanding Claim Types
- FHIR Implementation Guidlines
- Getting Help